Northstar.Client/mod/scripts/vscripts/ui/menu_ns_modmenu.nut

```diff
@@ -709,6 +709,15 @@ void function ReloadMods()
 	ClientCommand( "weapon_reparse" ) // weapon_reparse only works if a server is running and sv_cheats is 1, gotta figure this out eventually
 	SetConVarBool( "sv_cheats", svCheatsOriginal )
 
+	array<string> materialPaths = [ "cable", "correction", "debug", "decals", "dev", "effect", "engine", "models", "particle", "sprites", "tools", "ui", "vgui", "world" ]
+
+	foreach ( dir in materialPaths )
+	{
+		// Ex: "mat_reloadmaterial models/" will reload every vtf and vmt under the "models" path
+		// The reason this is done this way, is because Respawn removed "mat_reloadallmaterials" for some reason
```

Comment:
Would be good to look into how much this was removed (like just the command or the implementation as well)

Reply:
I looked into this, seems pretty gone. Might still exist in an interface somewhere but the only trace of the string at least is engine trying to call the concommand.